core/src/test/resources/edu/wpi/grip/projects/testALL.grip

```diff
@@ -405,9 +405,6 @@
       <grip:Input step="34" socket="1">
         <value>COLOR_BGR2GRAY</value>
       </grip:Input>
-      <grip:Input step="34" socket="2">
-        <value>0.0</value>
-      </grip:Input>

```

Comment:
So somewhere along the way we had a breaking change that we never did a major version bump for?